src/ui/tutorial/tutorial.js

```diff
@@ -24,16 +25,21 @@ goog.require('cwc.mode.Modder.Events');
 goog.require('cwc.mode.Type');
 goog.require('cwc.renderer.Helper');
 goog.require('cwc.soy.ui.Tutorial');
-goog.require('cwc.ui.Helper');
+goog.require('cwc.utils.Database');
+goog.require('cwc.utils.Helper');
 goog.require('cwc.utils.Events');
 goog.require('cwc.utils.Logger');
 goog.require('cwc.utils.mime.Type');
+goog.require('cwc.utils.Resources');
 
 goog.require('goog.dom');
+goog.require('goog.html.SafeHtml');
+goog.require('goog.html.sanitizer.HtmlSanitizer');
 goog.require('goog.events');
 goog.require('goog.soy');
+goog.require('goog.string');
 goog.require('goog.style');
-
+goog.require('soydata.VERY_UNSAFE');
```

Comment:
Please avoid the usage of this for user generated content outside of an sandbox, because its not very safe.

Reply:
It's necessary for code moved from https://github.com/google/coding-with-chrome/blob/b2316c086a8618242a55a8b21494a9d2b9b567dd/src/addon/workbench/workbench_project.js#L97 (already in master). That code could be rewritten to render inside a webview/iframe, but I wanted to move it with as few changes as possible first.